### DISCUSSION TITLE
CSS doesn't seem to be loaded properly

### DISCUSSION THREAD

**Discussion by djkeh** [mmistakes/so-simple-theme#353] [Q&A] [answered]
## Environment

- So Simple version: 3.2.0
- Ruby gem or remote theme version: 0.4.0 (jekyll-remote-theme)
- Jekyll version: 3.8.5
- Git repository URL: https://github.com/djkeh/djkeh.github.io
- GitHub Pages hosted (if yes provide URL to site): https://djkeh.github.io/
- Operating system: macOS mojave 10.14.6

## Expected behavior

My blog created via github pages shows normal screen without css issue.

## Steps to reproduce the behavior

No particular steps are needed. Prepare github pages, follow the guide(https://github.com/mmistakes/so-simple-theme#github-pages-method) and see what happens at the main page. If it's done already, just go to the main page. This is what actually happened to me today.

![image](https://user-images.githubusercontent.com/4949544/68192695-11126780-fff5-11e9-98d2-dca30c9e47df.png)

![image](https://user-images.githubusercontent.com/4949544/68193011-a7468d80-fff5-11e9-864b-5c71acb1cc75.png)

I commited a draft and some updates of my markdown documents and that was it. The things related to jekyll, its dependencies and so simple theme such as `Gemfile` or `_config.yml` are not changed at all. I got in to my blog too see the change of my article and I found this problem.

## Other

In my opinion, this maybe happened because so simple theme recently upgrade from `3.1.3` to `3.2.0`, but there's no release yet, so jekyll-remote-theme simply failed to bring all the resources which are required to show css properly.

Looking at the picture no. 2, I see my acutally rendered github pages have `/assets/css/main.css` resource annotated with ver. `3.2.0`, but according to the changelog of `3.2.0`, it's supposed to have `/assets/css/skins/default.css` as well to show skin feature, which is not there. I think this is the result of the lack of release `3.2.0`, and jekyll-remote-theme couldn't read `3.2.0`.

Another thing I checked is that if I follow ruby gem method(https://github.com/mmistakes/so-simple-theme#ruby-gem-method) and `bundle update`, `Gemfile.lock` shows the recent version of so-simple-theme is `3.1.3`. I think this is strange too. It feels like those are connected.

Please help. Thanks.

**Comment by mmistakes** [comment, marked as answer]
You're overriding the theme's default `_includes/head.html` file and it's missing some changes that load an addition stylesheet.

Add [this line](https://github.com/mmistakes/so-simple-theme/blob/master/_includes/head.html#L27) to your local `_includes/head.html` file.

```html
<link rel="stylesheet" href="{{ site.skin | default: '/assets/css/skins/default.css' | relative_url }}">
```

It's not mentioned in the docs yet, but you can lock a `remote_theme` version. You could roll back to `3.1.3` if need be. Remote theme's have the strange behavior of always pulling from `master`, so even though I may have not released an update those changes will be forced on you.

You'd install the theme like this:

```yaml
remote_theme: "mmistakes/so-simple-theme@3.1.3"
```

ref: https://github.com/benbalter/jekyll-remote-theme#declaring-your-theme

**Comment by djkeh** [comment]
Oh.. I missed that. totally forgot that I customized `head.html` long time ago. I did that for two reasons, one for `ROBOTS` related meta tag, and the other one for setting up my favicon.

Now I found the difference between my old `head.html` and so-simple-theme's one. Problem solved. Thanks to your help, @mmistakes!

**Comment by mmistakes** [comment]
Depending on what you changed in `head.html` you may find customizing [`_includes/head-custom.html`](https://github.com/mmistakes/so-simple-theme/blob/master/_includes/head-custom.html) more manageable.

It’s purposely left blank and meant to add your favicon and other things to the head. You won’t have to worry about merging in future changes since the theme will never change anything in this file. It’s more future proof than overriding `head.html` and hoping you catch any updates.

**Comment by djkeh** [comment]
I haven't noticed that either! This is beautiful. I'll try it right away, thanks!
